blocks/editable/test/index.js

```diff
@@ -10,6 +10,28 @@ import Editable from '../';
 import { diffAriaProps, pickAriaProps } from '../aria';
 
 describe( 'Editable', () => {
+	describe( 'Component', () => {
+		describe( '.getSettings', () => {
+			const value = [ 'Hi!' ];
+			const wrapper = shallow( <Editable value={ value } /> );
```

Comment:
This `wrapper` is used only in one tests. It would be better to put it there, because the other tests uses a different wrapper.